server/src/main/java/org/elasticsearch/transport/RemoteClusterAware.java

```diff
@@ -76,10 +75,9 @@ protected Map<String, List<String>> groupClusterIndices(Set<String> remoteCluste
         Set<String> clustersToRemove = new HashSet<>();
         for (String index : requestIndices) {
             // ensure that `index` is a remote name and not a datemath expression which includes ':' symbol
-            // since datemath expression after evaluation should not contain ':' symbol
-            String probe = IndexNameExpressionResolver.resolveDateMathExpression(index);
-            int i = probe.indexOf(RemoteClusterService.REMOTE_CLUSTER_INDEX_SEPARATOR);
-            if (i >= 0) {
+            boolean isDateMathExpression = (index.charAt(0) == '<' || index.startsWith("-<"));
```

Comment:
This seems to change semantics slightly so that we no longer assume a remote cluster definition if the expression says: `<abc:def`, where in the past, it would have to also have the closing `>`. This seems inadvertent?

I wonder if it was better to check for `-` as the first character and then probe the rest, just like the original code? It seems `-` is handled in this method (further down) already so seems appropriate.

Reply:
`<abc:def` can not be a valid remote expression since remote name can not include `<`. So I think not handling it as a remote should be correct?

Reply:
perhaps add a comment around `<` not being support as part of a remote cluster alias or index? I guess your point is that date math expressions are really the only scenario where this notation can be used?

Reply:
I did a quick test of trying to create a remote that starts with "<" and it failed both in the elasticsearch.yml file and via the `PUT _cluster/settings` API call, so it seems correct to say that remote cluster aliases cannot start with that character.